esmvalcore/preprocessor/_multimodel.py

```diff
@@ -399,23 +343,39 @@ def multi_model_statistics(products,
                            keep_input_datasets=True):
     """Compute multi-model statistics.
 
-    This function computes multi-model statistics on cubes or products.
-    Products (or: preprocessorfiles) are used internally by ESMValCore to store
+    This function computes multi-model statistics on a list of ``products``,
+    which can be instances of :py:class:`~iris.cube.Cube` or
+    :py:class:`~esmvalcore.preprocessor.PreprocessorFile`.
+    The latter is used internally by ESMValCore to store
     workflow and provenance information, and this option should typically be
     ignored.
 
-    This function was designed to work on (max) four-dimensional data: time,
-    vertical axis, two horizontal axes. Apart from the time coordinate, cubes
-    must have consistent shapes. There are two options to combine time
-    coordinates of different lengths, see the `span` argument.
+    Apart from the time coordinate, cubes must have consistent shapes. There
+    are two options to combine time coordinates of different lengths, see the
+    ``span`` argument.
+
+    Uses the statistical operators in :py:mod:`iris.analysis`, including
+    ``mean``, ``median``, ``min``, ``max``, and ``std``. Percentiles are also
+    supported and can be specified like ``pXX.YY`` (for percentile ``XX.YY``;
+    decimal part optional).
+
+    Notes
+    -----
+    Some of the operators in :py:mod:`iris.analysis` require additional
+    arguments. Except for percentiles, these operators are currently not
+    supported.
+
+    Lazy operation is supported for all statistics, except
+    ``median``, ``percentile``, ``gmean`` and ``hmean``.
```

Comment:
nice! we need to make sure this list gets updated when things change

Reply:
Good spot! Actually I forgot to remove this one from #968 , that's were all the lazy stuff is implemented. This PR just makes the code ready for that.